tools/bin/cldev

```diff
@@ -10,8 +10,6 @@ createdb chainlink_dev >/dev/null || true
 case "$1" in
     node | core | n)
       key='0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f'
-      echo "** Importing default key $key"
-      go run -ldflags "$LDFLAGS" ./core/main.go --  node import tools/secrets/$key.json >/dev/null
```

Comment:
@RyanRHall Is there a clean equivalent way of scripting this with the new keystore?